install/kubernetes/helm/istio/charts/pilot/values.yaml

```diff
@@ -14,6 +14,8 @@ sidecar: true
 traceSampling: 1.0
 # if protocol sniffing is enabled. Default to false.
 enableProtocolSniffing: false
+# timeout for protocol detection. Pilot will use default filter chain when times out
+protocolDetectionTimeout: 10ms
```

Comment:
Why would a user want to configure this? And the answer to this question should probably be included in the doc as well

Reply:
this goes into mesh config FYI.  And also to @howardjohn's point, copy the explanation from istio/api for the protocolDetectionTimeout